Sources/ComposableArchitecture/SwiftUI/Binding.swift

```diff
@@ -121,6 +121,139 @@ extension BindingState: CustomDebugStringConvertible where Value: CustomDebugStr
 
 extension BindingState: Sendable where Value: Sendable {}
 
+/// An action that describes simple mutations to some root state at a writable key path.
+///
+/// Used in conjunction with ``BindingState`` and ``BindableAction`` to safely eliminate the
+/// boilerplate typically associated with mutating multiple fields in state.
+///
+/// Read <doc:Bindings> for more information.
+public struct BindingAction<Root>: Equatable, @unchecked Sendable {
```

Comment:
Reorganized the file so that all the view stuff is at the bottom...makes reading the diff a lil more difficult but will point out the actual change.